config.json

```diff
@@ -559,8 +559,8 @@
       "slug": "flatten-array",
       "uuid": "2df8ed82-2a04-4112-a17b-7813bcdc0e84",
       "core": false,
-      "unlocked_by": "hello-world",
-      "difficulty": 1,
+      "unlocked_by": "series",
+      "difficulty": 3,
```

Comment:
Looks like this was merged in, unintended.

Reply:
I'm not sure what happened here but the merge commit doesn't include this change (thankfully). I merged in mobile so I might have missed something, but I was sure I saw this change had been reverted. It's weird Github still shows it here.

Reply:
Oh, good.

Reply:
I am seeing: 

https://github.com/exercism/ruby/blob/master/config.json

So it appears to me that it is there changed...

Reply:
Nope, it's still in `master` 😓 I'm going to revert this change in another PR.